apps/nextjs-website/src/components/organisms/Auth/SignUpForm.tsx

```diff
@@ -0,0 +1,354 @@
+                />
+              </Stack>
+              <Stack spacing={2} mb={2}>
+                <FormControl variant='outlined'>
+                  <InputLabel htmlFor='password-input' sx={{ top: '-8px' }}>
+                    {shared.password}
+                  </InputLabel>
+                  <OutlinedInput
+                    id='password-input'
+                    required
+                    type={showPassword ? 'text' : 'password'}
+                    onChange={(e) => setPassword(e.target.value)}
+                    onBlur={() => setIsPasswordDirty(true)}
+                    error={isPasswordDirty && !isPasswordValid}
+                    endAdornment={
+                      <InputAdornment position='end'>
+                        <IconButton
+                          aria-label='toggle password visibility'
+                          onClick={handleClickShowPassword}
+                          onMouseDown={handleMouseDownPassword}
+                          edge='end'
+                        >
+                          {showPassword ? <VisibilityOff /> : <Visibility />}
+                        </IconButton>
+                      </InputAdornment>
+                    }
+                    value={password}
+                    label={shared.password}
+                    inputProps={{
+                      sx: {
+                        padding: '8.5px 14px',
+                      },
+                    }}
+                  />
+                  <FormHelperText error={isPasswordDirty && !isPasswordValid}>
+                    {signUp.passwordPolicy}
+                  </FormHelperText>
+                </FormControl>
+              </Stack>
+              <Stack spacing={2} mb={2}>
+                <FormControl variant='outlined'>
+                  <InputLabel
+                    htmlFor='confirm-password-input'
+                    sx={{ top: '-8px' }}
+                  >
+                    {shared.confirmPassword}
+                  </InputLabel>
+                  <OutlinedInput
+                    id='confirm-password-input'
+                    type={showPassword ? 'text' : 'password'}
+                    required
+                    onChange={(e) => setConfirmPassword(e.target.value)}
+                    endAdornment={
+                      <InputAdornment position='end'>
+                        <IconButton
+                          aria-label='toggle confirm password visibility'
+                          onClick={handleClickShowConfirmPassword}
+                          onMouseDown={handleMouseDownConfirmPassword}
+                          edge='end'
+                        >
+                          {showConfirmPassword ? (
+                            <VisibilityOff />
+                          ) : (
+                            <Visibility />
+                          )}
+                        </IconButton>
+                      </InputAdornment>
+                    }
```

Comment:
It seems like this visibility button is not properly connected. I think this button should share state with the other visibility so both buttons change state when one of them are pressed. 

[Screencast from 09-11-2023 12:10:34.webm](https://github.com/pagopa/developer-portal/assets/24475319/f53cdd7b-e59b-4896-a112-db8795b00097)